src/directives/index.js

```diff
@@ -0,0 +1,344 @@
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-title.js"):
+     */
+    Vue.directive('t-title', {
+      bind(el, binding){
+        // get unique id
+        const unique_v_t_title_attr = createDirectiveObj({
+          el,
+          attr: 'g3w-v-t-title-id'
+        });
+        const value= binding.value;
+        const i18Fnc = binding.arg;
+        const handler = () =>{
+          const title = i18Fnc === 'plugin' ? tPlugin(value) : t(value);
+          el.setAttribute('title', title);
+          el.setAttribute('data-original-title', title)
+        };
+        handler();
+        setUnwatch({
+          id: unique_v_t_title_attr,
+          unwatch: vm.$watch(() => ApplicationState.lng, handler)
+        });
+      },
+      unbind(el){
+        unbindWatch({
+          attr:'g3w-v-t-title-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t.js"):
+     */
+    Vue.directive("t", {
+      bind (el, binding) {
+        const unique_v_t_attr = createDirectiveObj({
+          el,
+          attr: 'g3w-v-t-id'
+        });
+        setUnwatch({
+          id: unique_v_t_attr,
+          unwatch: prePositioni18n({
+            el,
+            binding,
+            i18nFnc: t
+          })
+        })
+      },
+      unbind(el){
+        unbindWatch({
+          el,
+          attr:'g3w-v-t-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-plugin.js"):
+     */
+    Vue.directive("t-plugin", {
+      bind (el, binding) {
+        const unique_v_t_plugin_attr = createDirectiveObj({
+          el,
+          attr: 'g3w-v-t-plugin-id'
+        });
+        setUnwatch({
+          id: unique_v_t_plugin_attr,
+          unwatch: prePositioni18n({
+            el,
+            binding,
+            i18nFnc: tPlugin,
+          })
+        })
+      },
+      unbind(el){
+        unbindWatch({
+          el,
+          attr: 'g3w-v-t-plugin-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-plugins.js"):
+     */
+    Vue.directive("plugins", {
+      bind(el) {
+        const showHideHandler = plugins =>{
+          el.classList.toggle('g3w-hide', plugins.length === 0)
+        };
+        showHideHandler(ApplicationState.plugins);
+        const unique_v_plugins_notify_attr = createDirectiveObj({
+          el,
+          attr: 'g3w-v-plugins-id'
+        });
+        setUnwatch({
+          id: unique_v_plugins_notify_attr,
+          unwatch: vm.$watch(() => ApplicationState.plugins, showHideHandler)
+        })
+      },
+      unbind(el){
+        unbindWatch({
+          el,
+          attr: 'g3w-v-plugins-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-online.js"):
```

Comment:
@volterra79 does it seem feasible to you?

Reply:
Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00